spec/system/move_tokens_spec.rb

```diff
@@ -35,17 +38,20 @@
 
   it "moves the token for other users" do
     map = create :map, zoom: 2
+    map.campaign.update(current_map: map)
     token = create :token, map: map, x: 0, y: 0
-    visit campaign_path(map.campaign)
-    find(".map-selector__option", text: map.name).click
 
+    visit campaign_path(map.campaign)
+    wait_for_connection
     using_session "other user" do
       visit campaign_path(map.campaign)
+      wait_for_connection
     end
 
     click_and_move_token(token, by: { x: 50, y: 50 })
 
     using_session "other user" do
+      wait_for_connection
```

Comment:
No need for the wait here as it was established at line 48

Reply:
Good catch, at one point I was getting unexpected failures and started overzealously adding it.

Reply:
It should only be a concern when visiting a new page then triggering a websocket based action on a separate session:

```ruby
visit some_path

using_session "other session" do
  visit some_path
end

do_something # can happen before "other session" connection is established
```